### PR TITLE
PureScript Icon

### PR DESCRIPTION
**Pull request by andys8**
#### Description

Adding the [PureScript](https://www.purescript.org) logo.

#### Requirements / Checklist

- [x] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/master/contributing.md)
- [x] Read or at least glanced at the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting)
- [x] Read or at least glanced at the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki)
- [ ] Scripts execute without error (if necessary):
  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`
- [ ] Extended the README and documentation if necessary, e.g. You added a new font please update the table

#### What does this Pull Request (PR) do?

* Add the PureScript SVG (https://github.com/purescript/logo)
* Modify the `original-source.otf` font with fontforge (`e530` is PureScript, OpenType CFF)
* Add it to the `font-patcher`

#### How should this be manually tested?

Have a deep look because I don't really know what I'm doing. Especially if I patched the font correctly (re-do it if in doubt) and if the correct unicode sequence is used in the `i_seti.sh` script. Thank you!

#### Any background context you can provide?

PureScript is a functional programming language like Elm or Haskell.

#### What are the relevant tickets (if any)?

To achieve this, I looked what a previous PR was doing: https://github.com/ryanoasis/nerd-fonts/pull/429
The changes should be pretty similar, but for PureScript

#### Screenshots (if appropriate or helpful)

screenshot-1607283979

### CODE REVIEW

File: bin/scripts/lib/i_seti.sh [hunk 1]
```diff
@@ -64,4 +64,5 @@ i='' i_custom_elm=$i
 i='' i_custom_elixir=$i
 i='' i_custom_electron=$i
 i='' i_custom_crystal=$i
+i='' i_custom_purescript=$i
```
Comment (andys8) [inline bin/scripts/lib/i_seti.sh:67]: Is this correct?

File: bin/scripts/lib/i_seti.sh [hunk 2]
```diff
@@ -64,4 +64,5 @@ i='' i_custom_elm=$i
 i='' i_custom_elixir=$i
 i='' i_custom_electron=$i
 i='' i_custom_crystal=$i
+i='' i_custom_purescript=$i
```
Comment (andys8) [inline bin/scripts/lib/i_seti.sh:67]: Please verify this is correct, thanks

**Comment by andys8** [comment]
@ryanoasis In comparison to other pull requests based on code, I'm a bit concerned with this one (editing a font), in case it's getting stale or merge conflicts have to be resolved. I'm not sure if everything was executed correctly in the first place. I'd appreciate any feedback a lot :)

File: bin/scripts/lib/i_seti.sh (hunk 2 above)
Comment (ryanoasis) [reply bin/scripts/lib/i_seti.sh:67]: Yep! that works

**Comment by ryanoasis** [comment]
Sorry for the long delay. This works well. thank you!

It's been a while to be honest since I've added any glyphs but looks like you made all the necessary changes, I tested it out briefly and works well.

We really should document adding glyphs to "core" but this PR is a good example 🤷🏻

**Pull request merged**